src/Maestro/tests/Maestro.ScenarioTests/ScenarioTests_Builds.cs

```diff
@@ -43,51 +45,61 @@ public Task DisposeAsync()
         public async Task ArcadeBuilds_EndToEnd()
         {
             TestContext.WriteLine("Darc/Maestro build-handling tests");
+            string scenarioDirectory = _parameters._dir.Directory;
 
             repoUrl = GetRepoUrl(repoName);
 
             // Create a build for the source repo
             Build build = await CreateBuildAsync(repoUrl, sourceBranch, sourceCommit, sourceBuildNumber, sourceAssets);
-            Build retrievedBuild = await MaestroApi.Builds.GetBuildAsync(build.Id);
+            Build retrievedBuild = await MaestroApi.Builds.GetBuildAsync(build.Id); 
             Assert.IsFalse(retrievedBuild.Released, "Retrieved build has Released set to true when it should be false");
 
-            //Release the build
+            // Release the build; this means gather-drop would not fetch anything unless --include-released were included in its args (which the next operation does)
```

Comment:
```suggestion
            // Release the build; this means gather-drop would not fetch anything unless --include-released was included in its args (which the next operation does)
```

Reply:
Will do, though I argue were is valid here in the [subjunctive / hypothetical sense](https://www.grammarly.com/blog/conditional-sentences-was-instead-of-were/), since it's not necessarily a given.